src/more-object-keys-entries.test.js

```diff
@@ -0,0 +1,60 @@
+import { getHouses, updateNumbers } from './more-object-keys-entries.js';
+
+
+test('it gets the names of the houses', () => {
+  const characters = [
+    {
+      name: 'Eddard',
+      spouse: 'Catelyn',
+      children: ['Robb', 'Sansa', 'Arya', 'Bran', 'Rickon'],
+      house: 'Stark'
+    },
+    {
+      name: 'Jon A.',
+      spouse: 'Lysa',
+      children: ['Robin'],
+      house: 'Arryn'
+    },
+    {
+      name: 'Cersei',
+      spouse: 'Robert',
+      children: ['Joffrey', 'Myrcella', 'Tommen'],
+      house: 'Lannister'
+    },
+    {
+      name: 'Daenarys',
+      spouse: 'Khal Drogo',
+      children: ['Drogon', 'Rhaegal', 'Viserion'],
+      house: 'Targaryen'
+    },
+    {
+      name: 'Mace',
+      spouse: 'Alerie',
+      children: ['Margaery', 'Loras'],
+      house: 'Tyrell'
+    },
+    {
+      name: 'Euron',
+      spouse: null,
+      children: [],
+      house: 'Greyjoy'
+    },
+    {
+      name: 'Jon S.',
+      spouse: null,
+      children: [],
+      house: 'Snow'
+    }
+  ]; //arrange
+  const output = getHouses(characters); //act
+  expect(output.sort()).toEqual(['Greyjoy', 'Snow', 'Arryn', 'Tyrell', 'Lannister', 'Targaryen', 'Stark'].sort()); //assert
```

Comment:
I mentioned this in the channel but you should change this line to: 
```
expect(output).toEqual(expect.arrayContaining(['Greyjoy', 'Snow', 'Arryn', 'Tyrell', 'Lannister', 'Targaryen', 'Stark']));
```
It's a fancy way of telling the test you want the things in the array but you don't care about the order of the things.